src/JuliusSweetland.OptiKey/Extensions/CharExtensions.cs

```diff
@@ -13,10 +13,10 @@ public static char ToUpperAndRemoveDiacritics(this char c)
             var cAsString = c.ToString(CultureInfo.InvariantCulture);
 
```

Comment:
Thank you - good spot! The whole method is unused as you say so I'll remove it after merging in your changes to clean up the source a tiny bit.